test/miner_state_channel_SUITE.erl

```diff
@@ -27,7 +27,8 @@
          conflict_test/1,
          reject_test/1,
          server_doesnt_close_test/1,
-         client_reports_overspend_test/1
+         client_reports_overspend_test/1,
+         dc_rewards_v4_test/1
```

Comment:
Can we rename this test? It's guarded behind v5 now. I'd suggest `missing_dc_reward_for_epoch`...

Reply:
sure